educator/src/Dscp/Educator/Config.hs

```diff
@@ -59,6 +63,11 @@ defaultEducatorConfig = upcast defaultWitnessConfig
     & sub #educator . sub #keys . sub #keyParams .~ defaultBaseKeyParams
     & sub #educator . sub #api . sub #botConfig . tree #params . selection ?~ "disabled"
     & sub #educator . sub #certificates . option #latex ?~ "xelatex"
+    & sub #educator . sub #certificates . option #downloadBaseUrl ?~ defBaseUrl
+    & sub #educator . sub #certificates . sub #issuer . option #id ?~ "Principal"
+  where
+    defBaseUrl = nothingToPanic "url is correct" $
+                 parseBaseUrl "https://localhost/api/certificates/v1/cert"
```

Comment:
Shouldn't it be `nothingToPanic "url is not correct"` or is this because you know it is correct already?
BTW this could be avoided entirely by building a `BaseUrl` instead of parsing one.

Reply:
Yeah, it means that I know that it is correct.
You're right about building the URL, but having it like this in the code (in a string literal) is easier to read, I guess.

Reply:
Good then, I figured that could be the motivation